src/index.ts

```diff
@@ -1,13 +1,10 @@
-import type { Annex as FissionAccountsAnnex } from "./components/account/implementation/fission-base.js"
-
 import * as Auth from "./auth.js"
 import * as Events from "./events.js"
 import * as Path from "./path/index.js"
 import * as Cabinet from "./repositories/cabinet.js"
 import * as CIDLog from "./repositories/cid-log.js"
 
 import { Query } from "./authority/query.js"
-import { Store } from "./common/crypto/store.js"
 import { Account, Agent, Channel, DNS, Depot, Identifier, Manners, Storage } from "./components.js"
```

Comment:
```suggestion
import { Account } from "./components.js"
```

Looks like after the changes we are only using `Account`.